src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -323,11 +323,18 @@ private static string[] SplitOnWhiteSpace(string s)
             return s.Split(null);
         }
 
-        private static bool ContainsSpecialCharacters(string element)
+        public static Char[] SpecialAndInvalidCharacters()
```

Comment:
The reason why it's not private is because this function is reused in the PublishPackageViewModel in order to display to the user the invalid characters.

Reply:
I see, thanks!

Reply:
Can we make it internal then?

Reply:
> Can we make it internal then?

Done.